docs/development.rst

```diff
@@ -96,10 +96,20 @@ Use production assets
 Front-end assets are compressed on production. To test these compressed assets
 locally, follow these steps:
 
-#. In .env, set ``DJANGO_DEBUG`` to ``False``
 #. Stop ``runserver`` if it's already running
+#. In .env, set ``DJANGO_DEBUG`` to ``False``
 #. Run ``python manage.py collectstatic``
-#. Run ``python manage.py runserver``
+#. Install `stunnel`_
+    * Mac: ``brew install stunnel``
+#. Run this from the root directory to generate a local cert and key for
+   stunnel (you can use the default values for all prompts):
+   ``openssl req -new -x509 -days 9999 -nodes -out stunnel/stunnel.pem -keyout stunnel/stunnel.pem``
+#. Run ``stunnel stunnel/dev_https``
+#. In another terminal, run ``HTTPS=1 python manage.py runserver 127.0.0.1:5000``
+#. Go to https://127.0.0.1:8443 and confirm the certificate exception
+#. Browse around the site at https://127.0.0.1:8443
+
+.. _stunnel: https://www.stunnel.org/index.html
```

Comment:
Can we change the title of this section from "Use production assets" to "Run in production mode" ?